src/parser/monk/mistweaver/modules/talents/ManaTea.js

```diff
@@ -162,17 +195,45 @@ class ManaTea extends Analyzer {
     };
   }
 
+  get avgOverHealing(){
+    let overHealingPercent = this.overhealing/(this.overhealing+this.effectivehealing);
+    overHealingPercent = Math.round(overHealingPercent * 1000)/1000;
+    return overHealingPercent;
+  }
+
+  get suggestionThresholdsMana(){
+    return {
+      actual: this.avgOverHealing,
+      isGreaterThan: {
+        minor: .20,
+        average: .30,
+        major: .40,
+      },
+      style: 'percentage',
+    };
+  }
+
   suggestions(when) {
     when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) => {
       return suggest(
         <>
-          Your mana spent during <SpellLink id={SPELLS.MANA_TEA_TALENT.id} /> can be improved. Always aim to cast your highest mana spells such as <SpellLink id={SPELLS.ESSENCE_FONT.id} /> or <SpellLink id={SPELLS.VIVIFY.id} />.
+          Your mana spent during <SpellLink id={SPELLS.MANA_TEA_TALENT.id} /> can be improved. Aim to only cast <SpellLink id={SPELLS.VIVIFY.id} /> untill the last second then cast <SpellLink id={SPELLS.ESSENCE_FONT.id} />.
         </>
       )
         .icon(SPELLS.MANA_TEA_TALENT.icon)
         .actual(`${formatNumber(this.avgMtSaves)} average mana saved per Mana Tea cast`)
         .recommended(`${(recommended / 1000).toFixed(0)}k average mana saved is recommended`);
     });
+    when(this.suggestionThresholdsMana).addSuggestion((suggest, actual, recommended) => {
+      return suggest(
+        <>
+          On average during your <SpellLink id={SPELLS.MANA_TEA_TALENT.id} /> you overhealed a decent amount. This can be improved. Always aim to cast into targets with lower health.
+        </>
+      )
+        .icon(SPELLS.MANA_TEA_TALENT.icon)
+        .actual(`${this.overHealingPercent*100} % average overhealing per Mana Tea cast`)
```

Comment:
there is a `formatPercentage()` helper function to format percentages for you.

Reply:
Didn't know this was a thing. I'll be on the look out for things like this in other files and then apply them in the future.

Reply:
Check in the `common` folder for a lot of these. It just keeps displays consistent.